MediaBrowser.Model/Extensions/ContainerExtensions.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using Jellyfin.Extensions;
+
+namespace MediaBrowser.Model.Extensions;
+
+/// <summary>
+/// Defines the <see cref="ContainerExtensions"/> class.
+/// </summary>
+public static class ContainerExtensions
+{
+    /// <summary>
+    /// Compares two containers, returning true if an item in <paramref name="inputContainer"/> exists
+    /// in <paramref name="profileContainers"/>.
+    /// </summary>
+    /// <param name="profileContainers">The comma-delimited string being searched.
+    /// If the parameter begins with the <c>-</c> character, the operation is reversed.</param>
+    /// <param name="inputContainer">The comma-delimited string being matched.</param>
+    /// <returns>The result of the operation.</returns>
+    public static bool ContainsContainer(this string? profileContainers, string? inputContainer)
```

Comment:
I'd prefer a static helper class instead of extensions since this only makes sense for 1 specific use-case and not strings in general.